package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "linkfree",
-  "version": "0.58.1",
+  "version": "0.62.3",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.58.1",
+      "version": "0.59.1",
```

Comment:
```suggestion
      "version": "0.58.1",
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix